src/utils/sdk.js

```diff
@@ -119,11 +119,21 @@ function expectedSDKVersion() {
   return version;
 }
 
+// Ensure that the user has a version of Xcode installed and usable.
 function ensureViableXCode() {
   const xcodeBuildExec = '/usr/bin/xcodebuild';
   if (fs.existsSync(xcodeBuildExec)) {
     const result = cp.spawnSync(xcodeBuildExec, ['-version']);
     if (result.status === 0) return;
+
+    const match = result.stdout
+      .toString()
+      .trim()
+      .match(/Xcode (\d+\.\d+)/);
+    const version = match ? match[1] : 0.0;
+    if (!semver.satisfies(version, '>14')) {
+      fatal(`Xcode version ${version} is not supported, please upgrade to Xcode 15 or newer`);
+    }
```

Comment:
```suggestion
    if (match && !semver.satisfies(semver.coerce(match[1]), '>14')) {
      fatal(`Xcode version ${version} is not supported, please upgrade to Xcode 15 or newer`);
    }
```

Looks like the version output can be formatted like '16.0', which needs to be coerced to a semver version.

Other suggestion in this change is to fall through to the fatal error below rather than using a fake `0.0` value, since the fatal error below is more descriptive.